src/Components/Content/index.js

```diff
@@ -11,9 +11,27 @@ function Content() {
         paddingTop: 40,
       }}
     >
+      {renderContent(selected)}
+      <Image
+        style={{ height: 200, width: 200 }}
+        source={require('./../../assets/contribute.png')}
+      />
       <Text>UNDER CONSTRUCTION</Text>
     </View>
   );
 }
 
+function renderContent(selected) {
+  const titles = [
+    'HOME',
+    'About us',
+    'Programs',
+    'Projects',
+    'Events',
+    'Contribute',
+  ];
```

Comment:
Maybe this is better in a common file since this array would be needed in multiple places?

Reply:
good point will move :)